content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/removal.rst

```diff
@@ -1,418 +1,166 @@
-use :doc:`units of measure
-</applications/inventory_and_mrp/inventory/product_management/product_replenishment/uom>`). Those
-three lots have the following numbers: :guilabel:`20001`, :guilabel:`20002`, and :guilabel:`20003`,
-each with five boxes in it.
-
-:guilabel:`20001` entered the stock on the 1st of July and expires on the 15th of July,
-:guilabel:`20002` entered on the 2nd and expires on the 14th of July, and :guilabel:`20003` entered
-on the 4th and expires on the 21st of July. A customer orders six boxes on the 5th of July. With
-the :abbr:`FEFO (First Expired, First Out)` strategy selected, a transfer is requested for the five
-boxes of :guilabel:`20002` and one from :guilabel:`20001`. The transfer for all the boxes in lot
-:guilabel:`20002` is because they have the closest expiration date. The transfer also requests one
-box from :guilabel:`20001` because has the next closest expiration date after lot
-:guilabel:`20002`.
-
-Basically, for every sales order of a product with the :abbr:`FEFO (First Expired, First Out)`
-strategy, a transfer is requested for the product that has the nearest expiration date from the
-order date.
-
-Closest Location
-----------------
-
-The *Closest Location* strategy is completely different from the other removal strategies. It is
-not related to the date of entry in the warehouse, but rather the location of the product. It is
-commonly used for products that do not deteriorate with time.
-
-The aim is to avoid making the warehouse worker take a long journey to the bottom of the stock when
-the product is also located at a near location. This method is only available if the
-:guilabel:`Storage Locations` setting is on. The closest location is actually the one that comes
-first in the alphabetic order.
-
-Use removal strategies
-======================
-
-To differentiate some units from others, the units need to be tracked, either by *lot* or by
-*serial number*. To do so, go to :menuselection:`Inventory --> Configuration --> Settings`. Then,
-activate the :guilabel:`Storage Location`, :guilabel:`Multi-Step Routes`, and :guilabel:`Lots &
-Serial Numbers` settings.
-
-.. image:: removal/enabled-features.png
-   :align: center
-   :alt: Removal strategy on a product category.
+.. admonition:: Priority
 
-To view all products with lots/serial numbers assigned to them, navigate to
-:menuselection:`Inventory app --> Products --> Lots/Serial Numbers`. This reveals a page with
-drop-down menus of all products assigned lots or serial numbers, filtered by *product* by default.
-To change the category these products are filtered by, click :guilabel:`Product` (in the search bar,
-in the top right of the page) to remove the default filter, and select a new filter if desired.
+   When there are different removal strategies applied on the location and product category for a
+   product, the *force* removal strategy set on the :guilabel:`Product Category` is applied.
```

Comment:
Don't know if I'd do a custom admonition here when an **Important** one seems to fit nicely here.

As well, I'd be *very* specific in this section in order to minimize misunderstandings.

```suggestion
.. important::
When there are different removal strategies applied on both the location and product category for a
product, the value set on the :guilabel:`Force Removal Strategy` field on the :guilabel:`Product
Category` form is applied as top priority.
```